_posts/2020-01-30-demystifying_samesite_cookie_updates/index.md

```diff
@@ -0,0 +1,129 @@
+
+In a Snowplow context, this means that your `network_userid` will work as it has always done. Tracking with first party cookies is our recommended practice, particularly as the end of the road is in sight for third party cookies in light of ITP changes in Safari and further restrictions by other browsers.
+
+#### Tracking with third party cookies
+
+Chrome 79 contains a number two flags which can be toggled to see if the SameSite updates will cause issues for a site. With Chrome 79, navigate to `chrome://flags` and enable `#same-site-by-default-cookies` and `#cookies-without-same-site-must-be-secure`. Restart the browser.
+
+Test your sites, with a focus on anything involving multiple domains or cross-site content. In particular you will want to pay attention to any analytics that involves tracking data stored inside cookies. If your site or tracking stops working as expected, turn each of the flags off one at a time (restarting the browser each time) to find out with setting is causing the issue.
+
+In a Snowplow context, you will want to check if your `network_userid` is tracking as expected. If the SameSite updates are causing issues for your configuratino, you should notice `network_userid`s being regenerated on every event that tracks for the same user (identified by `domain_userid` or `user_id`).
+
+### What you need to do about it
+
+For any cookies that need to be sent to a third party domain, you need to ensure that the cookie contains the `SameSite=None` and `Secure` attributes. This is going to depend on the vendor or software you are using that is making use of these cookies. In the case of Snowplow, the Snowplow Collector can be configured to return the cookie containing the `network_userid` field with the `SameSite=None` and `Secure` attribtues.
+
+Before thinking about taking steps to fix any third party cookies, it is worth considering if there is the option of moving to first party cookies by utilising running the vendor services on the same domain as your site. Where this is possible, it will not only solve any issues faced by the `SameSite` changes but will also ensure cookies will continue to work further into the future. Many browser vendors are also taking steps to prevent third party cookies working at all, see our [blog posts about ITP in Safari](https://snowplowanalytics.com/blog/2019/12/16/how-itp-2.3-expands-on-itp-2.1-and-2.2-and-what-it-means-for-your-web-analytics/) for more information.
+
+#### As a Snowplow Insights customer
+
+Snowplow Insights customers don't need to do anything with their collector configuration. The Snowplow team has already configured collectors to return the `network_userid` cookie with the `SameSite=None` and `Secure` attributes. It is worth checking that your tracking is being sent to a secure (HTTPS) endpoint as the new `Secure` attribute will lead to the cookie not being sent with insecure requests.
```

Comment:
"insecure" -> "non-secure"?

Reply:
That took more googling than I'd hoped. I think I'm going to go with unsecured requests.